.code-samples.meilisearch.yaml

```diff
@@ -690,7 +690,7 @@ get_all_keys_1: |-
 create_a_key_1: |-
   let mut key_options = KeyBuilder::new("Add documents: Products API key");
   key_options.with_action(Action::DocumentsAdd)
-      .with_expires_at("2042-04-02T00:42:42Z")
+      .with_expires_at(time::datetime!(2042-04-02 00:42:42 UTC))
```

Comment:
You don't need the quotes (like this: `time::datetime!("2042-04-02 00:42:42 UTC")`)?

Reply:
No, we don’t need quotes here because it’s a macro. So your date is checked at compile time directly, and `time` ensures it’s correct before running everything (and crashing at runtime).
BUT, the correct path is `time::macros::datetime` and not just `time::datetime` 😱 
Good catch!